SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/OutgoingMessages.h

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#pragma once
+
+#include <WorkerSDK/improbable/c_worker.h>
+
+enum class EOutgoingMessageType : int32
```

Comment:
Can we put these in a namespace or prefix them with `Spatial`?

Reply:
My preference for `improbable` namespace.

Reply:
I can put them in the `improbable` namespace. `Spatial` seems redundant in this case.